AmplifyPlugins/API/AWSAPICategoryPluginTests/AWSAPICategoryPluginTestBase.swift

```diff
@@ -65,4 +65,11 @@ class AWSAPICategoryPluginTestBase: XCTestCase {
             XCTFail("Error setting up Amplify: \(error)")
         }
     }
+
+    override func tearDown() {
+        if let api = apiPlugin {
+            api.reset {
+            }
```

Comment:
Consider adding a semaphore here to make sure this method doesn't complete until `reset` finishes

Reply:
I did consider putting a semaphore in the completion block, but I looked in the implementation of `reset`, and it behaves in a synchronous manner.  Seems like if the behavior of `reset` changes, and we fail to reset, then I prefer that our unit tests break.